american-sociological-association.csl

```diff
@@ -221,6 +231,20 @@
             <text macro="publisher"/>
           </group>
         </else-if>
+        <!--Software hack-->
+        <else-if type="book" variable="version" match="all">
+          <group prefix=" " delimiter=" ">
+            <group>
+              <text macro="title"/>
+              <text term="reference" prefix=" [" suffix="]." text-case="lowercase"/>
```

Comment:
Could you actually just hardcore computer program into the code and leave a Todo comment to localize once we have a proper term? That will travel better across languages than using a "wrong" redefined term.